src/components/form/form/form.stories.tsx

```diff
@@ -189,6 +204,16 @@ export const UsageWithFormik: Story = {
           { label: "Science", value: "science" },
         ],
       },
+      {
+        label: "Receive newsletter",
+        name: "subscribe_newsletter",
+        type: "radio",
+        required: true,
+        options: [
```

Comment:
The radio values is always an array (even though only one option can be selected). This is mostlikely due to the way how the serializer works. Lets have a look at that together.